content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Multiprotocol Shares"
+description: "Provides instructions on setting up SMB and NFSv4 mixed-mode shares."
+weight: 40
+tags:
+- scaleshares
+- scalesmb
+- scalenfs
+---
+
+{{< toc >}}
+
+## About Multiprotocol Shares
+
+A multiprotocol or mixed-mode NFS and SMB share supports both NFS and SMB protocols for sharing data.
+Multiprotocol shares allow clients to use either protocol to access the same data.
+This can be useful in environments with a mix of Windows systems and Unix-like systems, especially if some clients lack an SMB client.
+
+{{< hint type=tip >}}
+Carefully consider your environment and access requirements before configuring a multiprotocol share.
+For many applications, a single protocol SMB share provides better user experience and ease of administration. Linux clients can access SMB shares using [`mount.cifs`](https://linux.die.net/man/8/mount.cifs).
+{{< /hint >}}
+
+It is important to properly configure permissions and access controls to ensure security and data integrity when using mixed-mode sharing.
+To maximize security on the NFS side of the multiprotocol share, we recommend using NFSv4 and [Active Directory](#joining-active-directory) for Kerberos authentication.
+It is also important that NFS clients preserve extended attributes when copying files, or SMB metadata could be discarded in the copy.
+
+## Adding a Multiprotocol Share
+
+Adding a multiprotocol SMB and NFS share to your system involves several steps:
+
+1. [Start and configure](#starting-and-configuring-services) the SMB and NFS services.
+
+2. Join the TrueNAS server to an existing [Active Directory](#joining-active-directory) domain.
+   Configure a container, Kerberos admin, and user accounts in AD.
+
+3. [Set up a dataset](#creating-a-multiprotocol-share-dataset) for the new share with **Share Type** set to **Multiprotocol**.
```

Comment:
In the order of configuring shares, you first create the dataset, then the share so this is a preparation step or should come earlier in this overview procedure. If you create a preparation or first steps section, then you can mention this in the procedure intro, as in "After you take care of the first steps (link to it), then: and then list the steps for the overview. Leverage links to existing instructions like "AddSMBShare.md" and "AddingNFSShares.md" for the details. If only part of the instructions apply in this procedure, link to the sections that apply.

Reply:
The order is different here because you first need to configure the NFS service to work with KBR5, then join AD, then proceed with the typical order of creating the dataset and creating the shares.
 
The content here does leverage a lot from the existing articles, but the settings are unique enough that it isn't 1 to 1, plus we want to avoid bouncing people from page to page too much and make this a cohesive tutorial. The bottom section does link over because there are no unique instructions for connecting client systems once the share is set up.

Reply:
We could  look into snippetizing some sections to make updates easier, but this really needs to merge today (and be ported/merged to 23.10), so that might be an improvement to look at later this week.

Reply:
I think we can create a ticket to do the snippetizing and not worry about it here.